rpmlint/checks/IconSizesCheck.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class IconSizesCheck(AbstractCheck):
+    def __init__(self, config, output):
+        super().__init__(config, output)
+        self.file_size_regex = re.compile(r'/icons/[^/]+/(?P<x>\d+)x(?P<y>\d+)/')
+        self.info_size_regex = re.compile(r'(?P<x>\d+) x (?P<y>\d+)')
```

Comment:
These two could be class variables declared without init and then init itself can be dropped.